tests/ds_unselect_rules.sh

```diff
@@ -10,11 +10,11 @@ DS=$1
 UNSELECT_LIST=$2
 
 
-printf "Copying $DS file to /tmp\n"
+printf "Copying %s file to /tmp\n" "$DS"
```

Comment:
Please use single quotes for the first argument - that way you won't need to escape the backslash in `\n` - with double quotes, it should actually be `\\n`.